UI/app/src/main/java/com/example/pathfinder/Client/MqttClient.java

```diff
@@ -23,6 +23,15 @@ public void connect(String username, String password, IMqttActionListener connec
         options.setAutomaticReconnect(true);
         options.setCleanSession(true);
 
+
+        /**
+         * Last will topic - topic for subscribers to follow
+         * will_Retained - message will be retained bt the server, even after connection loss/ reconnection
+         */
+        String WILL_TOPIC = "/smartcar/connectionLost";
+        String PAYLOAD = "Connection unexpectedly lost";
```

Comment:
The will_Retained in this case is the String "PAYLOAD" at line 32? If so, is there a reason why it's called PAYLOAD? Just curious, maybe I don't get it in this context

Reply:
I've changed the documentation to be easier to understand, will push new changes soon.